script/upgrades/MU01/governance/Config.sol

```diff
@@ -354,4 +370,74 @@ library Config {
       config.minimumReports = 2;
     }
   }
+
+  function cEURUSDCConfig(
+    Contracts.Cache storage contracts,
+    uint8 phase
+  ) internal returns (PoolConfiguration memory config) {
+    require(phase >= 1 && phase <= 3, "phase must be 1, 2, or 3");
+    config = PoolConfiguration({
+      asset0: contracts.celoRegistry("StableTokenEUR"),
+      asset1: contracts.dependency("BridgedUSDC"),
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: false,
+      medianDeltaBreakerThreshold: FixidityLib.wrap(0),
+      medianDeltaBreakerCooldown: 0,
+      smoothingFactor: 0,
+      isValueDeltaBreakerEnabled: true,
```

Comment:
@sissnad Just double checking this is intentional. Buth new pools(USDCEUR & USDCBRL) are using the value delta breaker instead of the median delta breaker.

Reply:
We will update this when the config's ready